pkg/config/event.go

```diff
@@ -74,30 +75,81 @@ func readEnv(key string, def string) string {
 	return value
 }
 
+// SetupCloudEventsKafkaClient create the cloud events kafka client that will send events to the given kafka topic
+func SetupCloudEventsKafkaClient(topic string) (v2.Client, error) {
+	kafkaServers := strings.Split(LoadedConfig.Kafka.Bootstrap.Servers, ",")
+	saramaConfig, err := GetSaramaConfig()
+	if err != nil {
+		return nil, fmt.Errorf("error getting sarama config: %w", err)
+	}
+
+	topicTranslator := event.NewTopicTranslationWithClowder(clowder.LoadedConfig)
+	mappedTopicName := topicTranslator.GetReal(topic)
+
+	if mappedTopicName == "" {
+		mappedTopicName = topic
+	}
+
+	protocol, err := kafka_sarama.NewSender(kafkaServers, saramaConfig, mappedTopicName)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create kafka_sarama protocol: %w", err)
+	}
+
+	c, err := v2.NewClient(protocol, v2.WithTimeNow(), v2.WithUUIDs())
+	if err != nil {
+		return nil, fmt.Errorf("failed to create cloud events client: %w", err)
+	}
+	return c, nil
+}
+
+// SetupTemplatesNotifications creates the cloud events kafka client for sending events to the patch service
+func SetupTemplatesNotifications() {
```

Comment:
I think i'd perfer if we referred to these as events or messages internally.  Notifications is a console dot thing that is implemented via kafka events/messaging, but also had a rest implementation as well.

Reply:
(and to be clear, i think its fine to call kafka messages sent to platform.notifications.ingress as notifications, but calling it all generically 'notifications' might be confusing)

Reply:
agreed, it could be confusing. I renamed everything to "events".

Reply:
I also renamed `pkg/event -> pkg/kafka` and `pkg/notifications -> pkg/event`. I think these names better represent what the packages were being used for.